File: python/List-2/centered_average.py
def centered_average(nums):
  min = nums[0]
  max = nums[0]
  sum = 0
  for i in range(len(nums)):
    if(nums[i] < min):
      min = nums[i]
    if(nums[i] > max):
      max = nums[i]
  for i in range(len(nums)):
    sum += nums[i]
  return (sum - min - max)/(len(nums)-2)

File: java/Array-2/centeredAverage.java
public int centeredAverage(int[] nums) {
  int min = nums[0];
  int max = nums[0];
  int total = 0;
  for(int i=0; i<nums.length; i++){
    if(nums[i] < min){
      min = nums[i];
    }
    if(nums[i] > max){
      max = nums[i];
    }
  }
  for(int i : nums){
    total += i;
  }
  total -= min+max;
  return total/(nums.length-2);
}
